package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Node-Blog",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/bkwillia052/Node-Blog.git",
+  "author": "bkwillia052 <bkwillia052@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "server": "nodemon index.js"
+  },
+  "dependencies": {
+    "express": "^4.16.4",
```

Comment:
you're using cors in your application, but don't have it as a dependency in your package.json. You need to include all dependencies in your package.json, so other developers will get the right packages installed when they download your repository. This also is necessary for proper deployments

Reply:
also need to include morgan.

this should be taken care of when you do `yarn add cors` or `npm install cors`, so I'm not sure what your'e doing so that you didn't get these added to your dependencies?

Reply:
also helmet